src/blocks/content-helper/icons/views-icon.tsx

```diff
@@ -0,0 +1,21 @@
+/**
+ * WordPress dependencies
+ */
+import { SVG, Path } from '@wordpress/components';
+
+export const ViewsIcon = () => (
+	<SVG
```

Comment:
As above - `aria-hidden` is needed since this is purely visual decoration.

Reply:
Added in next line.